app/settings.py

```diff
@@ -66,7 +66,24 @@
 OSF_BASE_URL = env.OSF_BASE_URL.rstrip("/")
 OSF_API_BASE_URL = env.OSF_API_BASE_URL.rstrip("/")
 ALLOWED_RESOURCE_URI_PREFIXES = {OSF_BASE_URL}
-SESSION_COOKIE_DOMAIN = env.SESSION_COOKIE_DOMAIN
+# SESSION_COOKIE_DOMAIN = env.SESSION_COOKIE_DOMAIN
+SESSION_ENGINE = "django.contrib.sessions.backends.cache"
+SESSION_COOKIE_NAME = env.OSF_AUTH_COOKIE_NAME
+OSF_AUTH_COOKIE_SECRET = env.OSF_AUTH_COOKIE_SECRET
+REDIS_HOST = env.REDIS_HOST
+
+CACHES = {
+    "default": {
+        "BACKEND": "django.core.cache.backends.redis.RedisCache",
```

Comment:
So, what's the difference between `django_redis.cache.RedisCache` (which OSF uses) and `django.core.cache.backends.redis.RedisCache` and why we choose the latter for GV?

Reply:
`django.core.cache.backends.redis.RedisCache` is the native Django `redis` adapter while `django_redis.cache.RedisCache` is an addon called `django_redis`. I believe when we were refactoring sessions for OSF we use the `django_redis` addon because we thought it'd gives us some features that improves performance. I chose to go with the native one as we don't really need `djang_redis`.